public/admin/config.yml

```diff
@@ -445,3 +439,57 @@ collections:
           - label: Metadata description
             name: metadataDescription
             widget: text
+      - label: 'Products: Full Pages'
+        name: 'ProductsFullPages'
+        file: src/cms-content/learn/products-full-pages.json
+        fields:
+          - label: Product
+            name: product
+            widget: list
+            fields:
+              - {
+                  label: Product name,
+                  name: productName,
+                  widget: string,
+                  hint: 'Product name (ex. API)',
+                }
+              - {
+                  label: Product ID,
+                  name: productId,
+                  widget: string,
+                  hint: 'Must match ID used on products landing page, ie. product name all lowercase, separated by dashes if multi-work (ex. response-simulator)'
+                }
+              - {
+                  label: Product Intro,
+                  name: productIntro,
+                  widget: string,
+                  hint: 'Main intro before sections'
+                }
+              -
+                label: Sections
+                name: sections
+                widget: list
+                fields:
+                  - {
+                      label: Section title,
+                      name: sectionTitle,
+                      widget: string,
+                      hint: 'ex: Why our API'
+                    }
+                  - {
+                      label: Section ID,
+                      name: sectionId,
+                      widget: string,
+                      hint: 'Section title all lowercase, separated by dashes if multi-work (ex. why-our-api)'
+                    }
+                  - {
+                      label: Section Body,
+                      name: sectionBody,
+                      widget: markdown,
+                    }
+              - label: Metadata title
```

Comment:
If this metadata copy doesnt differ from that of the products landing page (?), ill just delete